### PR TITLE
W-13840863-EntitiesOverview-LimitedGA-SY

### PR DESCRIPTION
**Pull request by sarathecoubian**
# Writer's Quality Checklist

Before merging your PR, did you:

- [ ] Run spell checker
- [ ] Run link checker to check for broken xrefs
- [ ] Check for orphan files
- [ ] Perform a local build and do a final visual check of your content, including checking for:
  - Broken images
  - Dead links
  - Correct rendering of partials if they are used in your content
  - Formatting issues, such as:
    - Misnumbered ordered lists (steps) or incorrectly nested unordered lists
    - Messed up tables
    - Proper indentation
    - Correct header levels
- [ ] Receive final review and signoff from:
  - Technical SME
  - Product Manager
  - Editor or peer reviewer
  - Reporter, if this content is in response to a reported issue (internal or external feedback)
- [ ] If applicable, verify that the software actually got released

### CODE REVIEW

File: modules/ROOT/pages/entities-overview.adoc [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+* Response time 
+* Requests
+* Errors 
+* Client App (for APIs)
+* Locations (for APIs)
+* Org ID 
+
+You can filter entities with the following metrics:
+
+
```
Comment (sarathecoubian) [inline modules/ROOT/pages/entities-overview.adoc:39]: // need to add

File: modules/ROOT/pages/entities-overview.adoc [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+=== Setting the Time Period to Display
+
+Use the drop-down in the calendar icon to select the time period to display in a graph.
+
+Your setting applies across all charts in the entity summary. 
+
+=== Changing the Time Period to Display
+
+You can display an earlier or later time period in your chart, and you can zoom in or out to cover a shorter or longer period of time.
```
Comment (vimil-saju) [inline modules/ROOT/pages/entities-overview.adoc]: right now we don't support zoom-in/zoom-out functionality on the date picker.

File: modules/ROOT/pages/entities-overview.adoc [hunk 3]
```diff
@@ -0,0 +1,166 @@
...
+Total Requests::
+The sum of inbound and outbound requests according to your current filter conditions.
+
+Average Latency::
+[I don't know] , calculated in milliseconds. 
+
+Average Error Rate::
+The percentage of requests that result in errors according to your current filter conditions.
+
+Active Alerts::
```
Comment (vimil-saju) [inline modules/ROOT/pages/entities-overview.adoc]: we don't show Active Alerts for initial launch. We have Active Entities card instead.

File: modules/ROOT/pages/entities-overview.adoc (hunk 2 above)
Comment (sarathecoubian) [reply modules/ROOT/pages/entities-overview.adoc]:
```suggestion
You can display an earlier or later time period in your chart to cover a shorter or longer period of time.
```

File: modules/ROOT/pages/entities-overview.adoc (hunk 3 above)
Comment (sarathecoubian) [reply modules/ROOT/pages/entities-overview.adoc]:
```suggestion
Active Entities::
```

File: modules/ROOT/pages/entities-overview.adoc [hunk 4]
```diff
@@ -0,0 +1,166 @@
...
+The total number of alerts configured for entities according to your current filter conditions.
```
Comment (sarathecoubian) [inline modules/ROOT/pages/entities-overview.adoc]:
```suggestion
The total number of active entities according to your current search conditions.
```

File: modules/ROOT/pages/entities-overview.adoc [hunk 5]
```diff
@@ -0,0 +1,166 @@
...
+=== Performance Trends Charts
+
+Request Latency::
+[I don't know], calculated in milliseconds. 
+
+Error Volume::
+The total number of failed inbound and outbound requests to entities according to your current filter conditions.
+
+Request Volume:: 
+The total number of inbound and outbound requests to entities according to your current filter conditions.
```
Comment (sarathecoubian) [inline modules/ROOT/pages/entities-overview.adoc]:
```suggestion
The total number of inbound and outbound requests to entities according to your current search conditions.
```

File: modules/ROOT/pages/entities-overview.adoc [hunk 6]
```diff
@@ -0,0 +1,166 @@
...
+Active Alerts::
+The total number of alerts configured for entities according to your current filter conditions.
+
+=== Performance Trends Charts
+
+Request Latency::
+[I don't know], calculated in milliseconds. 
+
+Error Volume::
+The total number of failed inbound and outbound requests to entities according to your current filter conditions.
```
Comment (sarathecoubian) [inline modules/ROOT/pages/entities-overview.adoc]:
```suggestion
The total number of failed inbound and outbound requests to entities according to your current search conditions.
```

File: modules/ROOT/pages/entities-overview.adoc [hunk 7]
```diff
@@ -0,0 +1,166 @@
...
+The total number of entities according to your current filter conditions.
+
+Total Requests::
+The sum of inbound and outbound requests according to your current filter conditions.
+
+Average Latency::
+[I don't know] , calculated in milliseconds. 
+
+Average Error Rate::
+The percentage of requests that result in errors according to your current filter conditions.
```
Comment (sarathecoubian) [inline modules/ROOT/pages/entities-overview.adoc]:
```suggestion
The percentage of requests that result in errors according to your current search conditions.
```

File: modules/ROOT/pages/entities-overview.adoc [hunk 8]
```diff
@@ -0,0 +1,166 @@
...
+
+The *Entities Overview* page shows charts that detail entity performance. By default, the charts show data for entities in your current business group and environment. 
+
+=== Summary Charts
+
+Total Entities::
+The total number of entities according to your current filter conditions.
+
+Total Requests::
+The sum of inbound and outbound requests according to your current filter conditions.
```
Comment (sarathecoubian) [inline modules/ROOT/pages/entities-overview.adoc]:
```suggestion
The sum of inbound and outbound requests according to your current search conditions.
```

File: modules/ROOT/pages/entities-overview.adoc [hunk 9]
```diff
@@ -0,0 +1,166 @@
...
+
+
+== Understanding Summary and Performance Trends for Entities
+
+The *Entities Overview* page shows charts that detail entity performance. By default, the charts show data for entities in your current business group and environment. 
+
+=== Summary Charts
+
+Total Entities::
+The total number of entities according to your current filter conditions.
```
Comment (sarathecoubian) [inline modules/ROOT/pages/entities-overview.adoc]:
```suggestion
The total number of entities according to your current search conditions.
```

File: modules/ROOT/pages/entities-overview.adoc [hunk 10]
```diff
@@ -0,0 +1,166 @@
...
+// steal Setting Data Display Options in a Dashboard and Setting the Time Period Display from dashboards-using.adoc but change the titles.
+
+// see if Changing the Time Period to Display still applies.
+
+// steal Refreshing Data
+
+
+== Understanding Summary and Performance Trends for Entities
+
+The *Entities Overview* page shows charts that detail entity performance. By default, the charts show data for entities in your current business group and environment.
```
Comment (sarathecoubian) [inline modules/ROOT/pages/entities-overview.adoc]:
```suggestion
The *Entities Overview* page shows charts that detail entity performance. By default, the charts show data for entities in your current business group and environment. When you modify your search settings, the data in the charts changes accordingly.
```

File: modules/ROOT/pages/entities-overview.adoc [hunk 11]
```diff
@@ -0,0 +1,166 @@
...
+=== Summary Charts
+
+Total Entities::
+The total number of entities according to your current filter conditions.
+
+Total Requests::
+The sum of inbound and outbound requests according to your current filter conditions.
+
+Average Latency::
+[I don't know] , calculated in milliseconds.
```
Comment (sarathecoubian) [inline modules/ROOT/pages/entities-overview.adoc]: @vimil-saju Can you help with a definition here?

File: modules/ROOT/pages/entities-overview.adoc [hunk 12]
```diff
@@ -0,0 +1,166 @@
...
+Average Error Rate::
+The percentage of requests that result in errors according to your current filter conditions.
+
+Active Alerts::
+The total number of alerts configured for entities according to your current filter conditions.
+
+=== Performance Trends Charts
+
+Request Latency::
+[I don't know], calculated in milliseconds.
```
Comment (sarathecoubian) [inline modules/ROOT/pages/entities-overview.adoc]: @vimil-saju definition here too?

File: modules/ROOT/pages/entities-overview.adoc [hunk 13]
```diff
@@ -0,0 +1,166 @@
...
+
+You can display an earlier or later time period in your chart to cover a shorter or longer period of time.
+
+[cols="1,4"]
+|===
+| image:zoom-earlier-later.png[Zoom Out, Show Earlier and Later] a|
+To display earlier or later data, you can use the less-than (&lt;) and greater-than (&gt;) icons.
+
+To zoom out and display data over a longer time period, you use the magnifying glass icon located between the less-than and greater-than icons
+|===
```
Comment (sarathecoubian) [inline modules/ROOT/pages/entities-overview.adoc]:
```suggestion
```

File: modules/ROOT/pages/entities-overview.adoc (hunk 12 above)
Comment (sarathecoubian) [reply modules/ROOT/pages/entities-overview.adoc]: @sathishpaulleo cc

File: modules/ROOT/pages/entities-overview.adoc [hunk 14]
```diff
@@ -0,0 +1,192 @@
...
+=== Changing the Time Period to Display
+
+You can display an earlier or later time period in your chart to cover a shorter or longer period of time.
+
+
+// steal Setting Data Display Options in a Dashboard and Setting the Time Period Display from dashboards-using.adoc but change the titles.
+
+// see if Changing the Time Period to Display still applies.
+
+// steal Refreshing Data
```
Comment (sarathecoubian) [inline modules/ROOT/pages/entities-overview.adoc]:
```suggestion
```

**Pull request merged**